src/mono/netcore/sample/iOS/Program.csproj

```diff
@@ -29,7 +29,7 @@
   <UsingTask TaskName="MonoAOTCompiler"
              AssemblyFile="$(ArtifactsBinDir)MonoAOTCompiler\$(Configuration)\$(NetCoreAppCurrent)\MonoAOTCompiler.dll" />
 
-  <Target Name="BuildAppBundle" AfterTargets="CopyFilesToPublishDirectory">
+  <Target Name="BuildAppBundle" AfterTargets="Build">
```

Comment:
Why the switch to build?

Reply:
Otherwise this target is not called during library test build.